docs/api.rst

```diff
@@ -89,8 +89,8 @@ ensembles module
 
 subset module
 -------------
-
-.. automodule:: xclim.subset
-   :members:
-   :undoc-members:
-   :show-inheritance:
+.. warning::
+    Subsetting is now offered via `clisops`. The functions offered by clisops
+    will be described here once the subsetting functions API is made available.
+    For now, refer to their documentation here:
+    :doc:`clisops subset examples <clisops:notebooks/subset>`
```

Comment:
We could, but I feel a link to the clisops doc is already sufficient? Why should we present the doc of another package?

Reply:
This is true. I thought it might be interesting to link directly to their command API as xclim now virtually loads `clisops` to expose their API as ours.

Reply:
Oh! So this would be a 0.19-temporary thing? Then I agree.

Reply:
Yeah, that's what I envisioned. Who knows? Maybe it would be neat to have an API exposed only if `xclim[gis]` is installed?